Kong/docs.konghq.com
# Refactor app.js: rm unused code and move some content into new js files

### Review

@Kong/team-docs 

### Summary

* rm unused code
* add code comments
* move content into new js files

New  js files:
* editable-code-snippet
* navbar
* promo-banner

### Reason

Cleaner code structure and remove unused code. 

### Testing

Tested locally. Please verify the following:

* Banner (blue, top of page)
* On small screen size, toggle the three-line nav bar at the top right of the screen
* Enter value in editable code snippet and use "Enter" key on keyboard
* Click around!

### Looking Ahead

I do recommend that eventually we refactor our jQuery to vanilla JavaScript to increase performance (by up to 25X times!).
